### PR TITLE
Remove the bashism in postinst

### PR DESCRIPTION
**Pull request by luxas**
Trivial change.
Fixes: #170 

@mikedanese @errordeveloper PTAL

### CODE REVIEW

**Comment by errordeveloper** [comment]
How have you tested this? Clearly the error didn't come from the script you have changed, as it doesn't source anything and exec `configure` instead. Please take a closer look. I would much rather see a syntax fix and stick to plain `/bin/sh`.

**Comment by luxas** [comment]
@errordeveloper Now that's fixed. Please approve

File: debian/xenial/kubelet/debian/postinst [hunk 1]
```diff
@@ -34,8 +34,10 @@ esac
 
 mkdir -p /etc/kubernetes/manifests
 
+arch=$(uname -m);
 # kubelet is linked against /lib/ld-linux.so.3. On some systems, this symlink has to be present
-if [[ $(uname -m) == "arm"* && ! -f /lib/ld-linux.so.3 && -f /lib/ld-linux-armhf.so.3 ]]; then
+# ${arch#*arm} != ${arch} will be true if the $arch string contains the work arm, i.e. if it is an arm device
```
Comment (mikedanese) [inline debian/xenial/kubelet/debian/postinst]: s/work/word

**Comment by luxas** [comment]
@mikedanese fixed

**Comment by mikedanese** [comment]
lgtm

**Pull request merged**